shell/gpu/gpu_surface_metal.mm

```diff
@@ -160,7 +160,7 @@
   if (i != damage_.end()) {
     framebuffer_info.existing_damage = i->second;
   }
-
+  framebuffer_info.supports_partial_repaint = true;
```

Comment:
Do we need to ensure that this is an iOS device and not macOS for example? I realize that we typically go through the compositor api for macOS, but AFAIK there is a boolean flag we can flip to enable root surface rendering on macOS as well. I was wondering if there was a way to delegate this to the iOS specific impl.

Reply:
Right now this is only set in `GPUSurfaceMetal::AcquireFrameFromMTLTexture`, which is iOS specific. On MacOS we use `AcquireFrameFromMTLTexture`.